### PR TITLE
[No QA] Bump expensify-common

### PR DESCRIPTION
**Pull request by alex-mechler**
Hash bump for https://github.com/Expensify/expensify-common/pull/346

@thienlnam assigning you since you reviewed the original pr

### CODE REVIEW

**Comment by alex-mechler** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by alex-mechler** [comment]
recheck

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "@react-native-community/async-storage": "^1.12.1",
-    "expensify-common": "git+https://github.com/Expensify/expensify-common.git#485113217cd319cd3b2185e14a5fd03d36b8d3e7",
+    "expensify-common": "git+https://github.com/Expensify/expensify-common.git#09e7a7014537b4fec527ea12039eb227a72133fd\n",
```
Comment (thienlnam) [inline package.json:16]: also contains '\n'

**Review by thienlnam — CHANGES_REQUESTED** [review]

**Comment by alex-mechler** [comment]
Updated

**Review by thienlnam — APPROVED** [review]

**Pull request merged**